src/include/storage/tile.h

```diff
@@ -13,6 +13,10 @@
 #pragma once
 
 #include <mutex>
+#include <vector>
```

Comment:
Doesn't look like these includes are needed in tile.h. Can you just put them in dictionary_encoding_tile.h?